corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/utils/SRLLink.java

```diff
@@ -0,0 +1,56 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes.utils;
+
+import java.text.ParseException;
+
+import edu.illinois.cs.cogcomp.core.datastructures.IntPair;
+
+/**
+ * This class captures data about the linkages between this node
```

Comment:
"linkages between this node"? Can you clarify this plz?

Reply:
The relationship between the verb and it's arguments... Not sure what vocabulary I should use for this...